### DISCUSSION TITLE
New to MD-Analysis

### DISCUSSION THREAD

**Discussion by Syedmahmood777** [MDAnalysis/mdanalysis#4821] [Introductions]
Hi everyone,
My name is Syed and I am an aspiring Data Scientist,
Looking forward to contribute to this repo.

I am a bit confused about whether or not I need domain related knowledge to contribute? 

I hope to be helpful and make valuable contributions.

**Comment by RMeli** [comment]
Hi @Syedmahmood777, welcome to MDAnalysis!

Everyone can contribute to the project, but depending on what you are planning to contribute to, some domain knowledge will be required. I would suggest to get started with the [User Guide](https://userguide.mdanalysis.org/stable/index.html).

**Reply by Syedmahmood777** [reply]
Hey @RMeli , thanks for responding!

I will look at the user guide and try to contribute as much as possible.

**Comment by chirag1701** [comment]
HI everyone
Myself Chirag Sindhwani, BTech student in Electrical Engineering at IIT BHU Varanasi. I have experience working with graph neural networks in the past, and I am currently working on a research project on equivariant graph networks for atomic simulations . I use a lot of MD simulations data and interested in contributing to the efforts of MDAnalysis team.  How should i start contributing ?
Thanks !

**Reply by orbeckst** [reply]
I'd say use MDAnalysis for your own work. Participate in the forums (discussions, discord). Then figure out which aspects of MDAnalysis are most important for you and have a look at the [open issues](https://github.com/MDAnalysis/mdanalysis/issues). It's not necessarily straightforward to just see which issues are easy to do and which ones require more insights, so it will be a lot easier if you gain some experience with MDAnalysis as a tool.

**Comment by chirag1701** [comment]
Thanks a lot! @orbeckst . I’ll start using MDAnalysis more in my work, join the discussions, and look into the open issues to see where I can pitch in.
